src/app/(proper_react)/(redesign)/(authenticated)/admin/feature-flags/page.tsx

```diff
@@ -99,7 +73,35 @@ export default async function FeatureFlagPage() {
     );
   };
 
+  const DeletedFlagsTable = (featureFlags: { data: Array<FeatureFlagRow> }) => {
+    const { data } = featureFlags;
+
+    if (!data || data.length === 0) {
+      return <p>No data</p>;
+    }
+
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Deleted At</th>
+          </tr>
+        </thead>
+        <tbody>
+          {data.map((item) => (
+            <tr key={item.name}>
+              <td>{item.name}</td>
+              <td>{item.deleted_at?.toString()}</td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
+    );
+  };
+
   const featureFlags = (await getAllFeatureFlags()) || null;
+  const deletedFeatureFlags = (await getDeletedFeatureFlags()) || null;
```

Comment:
Nit, but a good habit to build. Since these aren't returning a boolean:

```suggestion
  const featureFlags = (await getAllFeatureFlags()) ?? null;
  const deletedFeatureFlags = (await getDeletedFeatureFlags()) ?? null;
```